modules/cli-administrator-security-policy.adoc

```diff
@@ -49,6 +49,11 @@ $ oc adm pod-network isolate-projects project1 project2
 
 Manage roles and policies on the cluster.
 
+[NOTE]
```

Comment:
I see @bergerhoffer wanted to move this note above but the `policy` command is a bit broader than just SCCs so I would probably keep it next to the command as it was originally.

Also, instead of pointing out that "administrators create rolebindings", I'd rather rephrase this so that it says that the command is working differently starting version 4.5 and that it is that command that creates the rolebindings, which is the preferred way to handle access to the SCCs

Reply:
@stlaz: Changes made. Please let me know what you think.